concourse/model/traits/component_descriptor.py

```diff
@@ -297,16 +279,16 @@ def validation_policies(self):
 
     @property
     def ocm_repository(self) -> ocm.OciOcmRepository:
-        ocm_repo = self.raw.get('ocm_repository') or self.raw.get('ctx_repository')
-        # XXX hack for unittests
-        if not self.cfg_set:
-            return None
+        ocm_repo = self.raw.get('ocm_repository')
 
         # ocm_repo might be:
-        # - a cfg-element-name (type: CtxRepositoryCfg)
         # - a ocm-repository-url (type: str)
         # - a OCM Repository (type: dict / gcm.OcmRepository
 
+        # XXX hack for unittests
+        if not self.cfg_set:
+            return None
+
         if ocm_repo:
             try:
                 ctx_repo_cfg = self.cfg_set.ctx_repository(ocm_repo)
```

Comment:
I wonder, you removed the comment "a cfg-element-name (type: CtxRepositoryCfg)" above, but kept resolving `ocm_repo` as such. Please keep this consistent.

Reply:
good catch. Will rm the resolving-logic as well (it did not prove to be a good idea to have an indirection via cfg-element for ocm-repository-definition (this is one of the few cases where it is preferable IMO to have redundant configuration requiring explicit changes, if desired).

Reply:
done